src/Network/Receive/ServerType0.pm

```diff
@@ -837,7 +837,11 @@ sub items_nonstackable {
 	) {
 		return $items->{type7};
 	} elsif ($args->{switch} eq '0B0A') { # item_list
-		return $items->{type8};
+		if (grep { $masterServer->{serverType} eq $_ } qw(iRO_Renewal)) {
```

Comment:
Slick way to fix the iRO Renewal vs ALL the other server issue.   This will leave a good base if we have other servers that follow the same packet changes, by just inserting them in the qw()